### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by Andileh**
Please review my PR

### CODE REVIEW

File: WaterLog_Backend/Controllers/LocationsController.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        readonly IConfiguration _config;
+        public LocationsController(DatabaseContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+
+        // GET api/values
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<LocationsEntry>>> Get()
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/LocationsController.cs]: Get? Get what? Have a more descriptive function name

File: WaterLog_Backend/Controllers/LocationsController.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        {
+            return await _db.Locations.ToListAsync();
+        }
+
+        // GET api/values/
+        [HttpGet("{id}")]
+        public async Task<ActionResult<LocationsEntry>> Get(int id)
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/LocationsController.cs]: Again be more descriptive

Comment (FredJEhlers) [reply WaterLog_Backend/Controllers/LocationsController.cs]: The rest as well

File: WaterLog_Backend/Controllers/LocationsController.cs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                _db.Locations.Remove(entry);
+                await _db.SaveChangesAsync();
+            }
+            catch (Exception error)
+            {
+                throw new Exception(error.Message);
+            }
+        }
+    }
+}
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/LocationsController.cs]: empty line at EOF

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Controllers/LocationsController.cs (hunk 1 above)
Comment (Andileh) [reply WaterLog_Backend/Controllers/LocationsController.cs]: Here you getting values of the Location api. I changed the name to be GetLocations

File: WaterLog_Backend/Controllers/LocationsController.cs (hunk 2 above)
Comment (Andileh) [reply WaterLog_Backend/Controllers/LocationsController.cs]: I changed the rest as well

**Review by FredJEhlers — APPROVED** [review]

**Review by CeeyarRbt — APPROVED** [review]

**Pull request merged**